forge/routes/api/project.js

```diff
@@ -478,6 +479,19 @@ module.exports = async function (app) {
             changesToPersist.stack = { from: request.project.stack, to: stack }
         }
 
+        // Launcher settings
+        if (request.body?.launcherSettings?.healthCheckInterval) {
+            const oldInterval = await request.project.getSetting(KEY_HEALTH_CHECK_INTERVAL)
+            const newInterval = parseInt(request.body.launcherSettings.healthCheckInterval, 10)
+            if (isNaN(newInterval) || newInterval < 1000) {
+                reply.code(400).send({ code: 'invalid_heathCheckInterval', error: 'Invalid heath check interval' })
```

Comment:
The UI text says a value below 5000 is not recommended - I'd suggest we apply that as a minimum value here as well, and add some UI feedback that prevents a smaller value from being submitted.

Reply:
Yes, I did set an absolute minimum (safe guard), the other number was a recommendation. 

Will apply 5s as the hard minimum though.

Reply:
limit applied in [89db3d2](https://github.com/FlowFuse/flowfuse/pull/3716/commits/89db3d2acc9888145cb05d07b0d4e748ac899b55)

checking alert suitability (will adjust if not suitable)

Reply:
@knolleary Added specific alert (commit: [9412788](https://github.com/FlowFuse/flowfuse/pull/3716/commits/9412788ca8b7846b411376262ed2276b3ac37a3f) ) but it kinda makes part of the description field (highlighted in yellow) a bit redundant 

![image](https://github.com/FlowFuse/flowfuse/assets/44235289/7e4bb6ea-1f01-4469-b788-7e25fda052ae)

Reply:
I was more thinking about validation before the user can click save as we do elsewhere, for example, the username field:

<img width="290" alt="image" src="https://github.com/FlowFuse/flowfuse/assets/51083/359737f5-93c7-4800-8f24-20552b3e49e3">

Reply:
Added client side validation 

![image](https://github.com/FlowFuse/flowfuse/assets/44235289/20e57adf-d6ab-47f1-b1f7-9b0a93f864c7)


commit incomming.

Reply:
Added in [8378317](https://github.com/FlowFuse/flowfuse/pull/3716/commits/8378317c63bd7fa6d78b4a18e0997556f9b9cdf8)

Also adds e2e test in [81588d2](https://github.com/FlowFuse/flowfuse/pull/3716/commits/81588d21c592b3145107aee4c7bf4c4f570f6306)

```
FlowFuse - Instance - Settings - Launcher
    Validates health check interval user input
```